starcheck/server.py

```diff
@@ -69,20 +74,29 @@ def handle(self):
                 exc = traceback.format_exc()
 
         resp = json.dumps({"result": result, "exception": exc})
-        # print(f"SERVER send: {resp}")
+        logger.debug(f"SERVER send: {resp}")
 
         self.request.sendall(resp.encode("utf-8"))
 
 
 def main():
     global KEY
-
+    
     # Read a line from STDIN
     port = int(sys.stdin.readline().strip())
     KEY = sys.stdin.readline().strip()
-
-    print("SERVER: starting on port", port)
-
+    loglevel = sys.stdin.readline().strip()
+    
+    logmap = {'0': logging.CRITICAL,
```

Comment:
With regard to logging and "not worth it initially" I just ran with including the pass-throughs for verbose logging because it was easier to test.